tests/foreman/cli/test_user.py

```diff
@@ -1657,6 +1681,42 @@ def test_list_user_4(self, test_data):
             'id': user['id'],
             'email': user['email']} in result.stdout)
 
+    @skip_if_bug_open('bugzilla', 1204667)
+    @data(
+        {'mail': gen_string("latin1", 10) + "@somemail.com"},
+        {'mail': gen_string("utf8", 10) + "@somemail.com"}
+    )
+    def test_bugzilla_1204667(self, test_data):
+        """@Test: List User for utf-8,latin variations of Email Address
+
+        @Feature: User - list
+
+        @Steps:
+        1. Create User with above Email Address variations in [1] using valid
+        valid Username, First Name, Surname, Language, authorized by
+        2. List User
+
+        @Assert: User is listed
+
+        @BZ: 1204667
+
+        """
+        try:
+            user = make_user(test_data)
+        except CLIFactoryError as err:
+            self.fail(err)
+        self.__assert_exists(user)
```

Comment:
@Ichimonji10  there is already a method https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/cli/test_user.py#L33 written i just used it